hal/inc/hal_platform.h

```diff
@@ -246,4 +246,8 @@
 #define HAL_PLATFORM_NFC 0
 #endif /* HAL_PLATFORM_NFC */
 
+#ifndef HAL_INCREASE_CHARGING_CURRENT_WHEN_POWERED_BY_VUSB
+#define HAL_INCREASE_CHARGING_CURRENT_WHEN_POWERED_BY_VUSB (0)
```

Comment:
This sounds confusing as `VUSB` usually comes from the USB port, let's change this to `WHEN_POWERED_BY_VIN`.